webpack.config.devtools.js

```diff
@@ -8,7 +8,7 @@ webpackConfig.output.library = "Debugger";
 webpackConfig.plugins = [
   new webpack.DefinePlugin({
     "process.env": {
-      NODE_ENV: JSON.stringify("DEVTOOLS_PANEL"),
+      NODE_ENV: JSON.stringify("production"),
```

Comment:
Hm, I do see that using NODE_ENV in general is conflating it with the production meaning that libraries use. Because we load in the config below, and I can still just do `webpack --config webpack.config.devtools.js`, we can just use the config as the flag in `isFirefoxPanel` and never use NODE_ENV. That way NODE_ENV can be used normally: "production" turns on optimizations in libraries.

It might be nice to do turn on dev mode in the firefox panel build if we want to debug something there, but I'm ok forcing production here.